ecorise-react-app/src/Dashboard/index.js

```diff
@@ -0,0 +1,313 @@
+import React from "react";
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import Sidebar from '../shared-components/SideBar';
+import { FaUsers, FaTshirt, FaStar, FaProductHunt, FaPray, FaTruckPickup, FaMoneyBill, FaMonero } from "react-icons/fa";
+import "./style.css";
+import { DiResponsive } from "react-icons/di";
+import { useNavigate } from "react-router-dom";
+import { format } from "date-fns";
+import { usePickups, useMarkets, useRewards, useProducts,useUsers,usePayment } from "../Hook/ecoriseUseData";
+
+function Dashboard() {
+  const navigate = useNavigate();
+  const recycledPercentage = 70;
+
+  const { data: pickups = [], loading: pickupsLoading } = usePickups();
```

Comment:
The initial state of data in your hooks is an array. Why would you redeclare it here again?